Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -942,9 +943,18 @@ eval OpenSettings state = do
 eval (SearchExpireCountDown seconds id status timerID) state = do
   if status == "EXPIRED" then do
     _ <- pure $ clearTimer timerID
+    let tipViewData = HomeScreenData.initData.props.tipViewProps
+    _ <- pure $ setTipViewData (TipViewData { value : tipViewData })
     continue state { props { searchExpire = seconds } }
   else
-    continue state { props { searchExpire = seconds } }
+    case state.props.currentStage of
+      FindingQuotes -> continue state { props { searchExpire = seconds ,timerId = timerID , tipViewProps {isVisible = if state.props.customerTip.enableTips && (seconds <=139 || state.props.tipViewProps.isVisible) && (state.props.customerTip.tipActiveIndex >0) then true else false}} }
```

Comment:
instead of performing same action for two cases use if condition in the following manner : 

if any ( _ == state.props.currentStage) [FindingQuotes , QuoteList] then continue state { props { searchExpire = seconds ,timerId = timerID , tipViewProps {isVisible = if state.props.customerTip.enableTips && (seconds <=139 || state.props.tipViewProps.isVisible) && (state.props.customerTip.tipActiveIndex >0) then true else false}} }

else do
         _ <- pure $ clearTimer timerID
         let tipViewData = HomeScreenData.initData.props.tipViewProps
         _ <- pure $ setTipViewData (TipViewData { value : tipViewData })
         continue state { props { searchExpire = (getSearchExpiryTime "LazyCheck") ,timerId = timerID , tipViewProps {isVisible = false}} }

Reply:
done